CMakeLists.txt

```diff
@@ -358,6 +372,7 @@ ADD_SUBDIRECTORY(writeengine/splitter)
 ADD_SUBDIRECTORY(storage-manager)
 ADD_SUBDIRECTORY(datatypes)
 ADD_SUBDIRECTORY(tests)
+ADD_SUBDIRECTORY(jit)
```

Comment:
This directory must be moved into ./utils and the paths changed accordingly.

Reply:
Done.